app/controllers/miq_task_controller.rb

```diff
@@ -445,19 +447,17 @@ def get_layout
   end
 
   def get_session_data
-    @layout        = get_layout
-    @jobs_tab      = session[:jobs_tab] if session[:jobs_tab]
+    super
+    @layout = get_layout
+    @tasks_tab     = session[:tasks_tab] if session[:tasks_tab]
     @tabform       = session[:tabform]  if session[:tabform]
-    @lastaction    = session[:jobs_lastaction]
-    @tasks_options = session[:tasks_options] || ""
```

Comment:
Looking at `@tasks_options` .. it seems like this is only used in miq_task related files .. *and* ops_controller, as you correctly noticed.

But, in ops_controller, it's just dead :). (not used in application controller, nor any view other than under views/miq_task/`, nor any mixin...)

So.. given that, I guess it no longer makes sense to move it to the generic mixin, can you just keep it here instead?